core/java/src/org/openda/blackbox/wrapper/BBStochModelInstance.java

```diff
@@ -93,14 +93,14 @@ public IObservationOperator getObservationOperator(){
 	 * For each constraintExchangeItemId for which there is a range validation constraint this map contains
 	 * the corresponding RangeValidationConstraint object.
 	 */
-	private Map<String, RangeValidationConstraint> constraintExchangeItemIdConstraintMap = new LinkedHashMap<String, RangeValidationConstraint>();
+	private Map<String, RangeValidationConstraint> constraintExchangeItemIdConstraintMap = new LinkedHashMap<>();
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)